BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -355,6 +360,45 @@ class GalleryViewController: UIViewController {
     )
   }
   func openGoal(_ goal: Goal) { coordinator?.showGoal(goal) }
+
+  @objc private func keyboardWillShow(_ notification: Notification) {
```

Comment:
I would have thought that perhaps the bodies of these two `keyboardWillShow` and `keyboardWillHide` would have been the same block of code and a round of DRY cleansing could have been applied. I see though they are (currently) not identical.